lib/node_modules/@stdlib/stats/base/dists/levy/cdf/benchmark/benchmark.native.js

```diff
@@ -0,0 +1,73 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var bench = require( '@stdlib/bench' );
+var Float64Array = require( '@stdlib/array/float64' );
+var randu = require( '@stdlib/random/base/randu' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var tryRequire = require( '@stdlib/utils/try-require' );
+var pkg = require( './../package.json' ).name;
+
+
+// VARIABLES //
+
+var cdf = tryRequire( resolve( __dirname, './../lib/native.js' ) );
+var opts = {
+	'skip': ( cdf instanceof Error )
+};
+
+
+// MAIN //
+
+bench( pkg + '::native', opts, function benchmark( b ) {
+	var len;
+	var mu;
+	var x;
+	var c;
+	var y;
+	var i;
+
+	len = 100;
+	x = new Float64Array( len );
+	mu = new Float64Array( len );
+	c = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = ( randu() * 20.0 ) - 10.0;
```

Comment:
@AadishJ We can directly create arrays of random numbers from the desired distribution via `uniform` here for better clarity and to avoid the for-loop.